mass_mailing_partner/models/mail_mass_mailing_contact.py

```diff
@@ -40,30 +40,38 @@ def _onchange_partner_mass_mailing_partner(self):
 
     @api.model
```

Comment:
I see that your optimization is done specifically for importing contacts, which usually happens with a big list of records.

Have you considered taking this opportunity to migrate this method to `@api.model_create_multi`?

Reply:
What about this?

Reply:
Ok, done